utils/appliance/db.py

```diff
@@ -137,7 +137,12 @@ def setup(self, **kwargs):
         if self.appliance.version != LATEST:
             # We only execute this on downstream appliances.
             # TODO: Handle external DB setup. Probably pop the db_address and decide on that one.
-            self.enable_internal(**kwargs)
+            self.enable_internal(
```

Comment:
do we have some kind of safe-call wrapper?
at least i'd prefer one to duplicating the defaults